addons/io/org.openhab.io.neeo/web/js/thirdparty/filesaver.js

```diff
@@ -1,5 +0,0 @@
-
```

Comment:
This file should not be removed.

Reply:
Hi Christoph, 
That file has nothing to do with this binding but for some reason appeared in an earlier commit.
That is the reason this has now been deleted, was that the wrong way of dealing with this?

I believe that the updates I have made to this binding comply with the DCO.
So as soon as we are happy with the handling of the file above, we should be good to go.

Reply:
If this file does not belong to the changes please remove - or re-add - it from/to this PR. In other words do not delete it in this PR.
In case the file is wrong in the repository please create a separate PR.

Reply:
Hi Christoph,
Sorry to be a pain but despite an extensive search through Google I cannot find out how to remove or re-add the file.
I have found a few posts suggesting command line (RM) but they only appear to work if its the newest commit. I'm not even sure how/where to enter the commands, if they are correct!
I am using Windows 10 with a non functioning eclipse; so if you can direct me to a guide or help file, I will be very grateful.
Hopefully there is a simple solution as I can't see how to get out of the mess this file is causing, especially as I cannot carry out any kind of rollback in eclipse.

Reply:
Here is a [list of Windows based Gui clients](https://git-scm.com/download/gui/windows). If that does not work I suggest to close this PR and create a new one. Feel free to go the easiest way for you.